nginx/datadog_checks/nginx/nginx.py

```diff
@@ -105,6 +105,17 @@ def check(self, instance):
 
             for endpoint, nest in plus_api_chain_list:
                 response = self._get_plus_api_data(url, plus_api_version, endpoint, nest)
+
+                try:
+                    if isinstance(response, dict):
+                        version_plus = response.get('nginx_version')
+                    else:
+                        version_plus = json.loads(response).get('nginx_version')
+
+                    self._set_version_metadata(version_plus)
```

Comment:
Should this be in a loop?

Reply:
Updated to get the version from an explicit endpoint